style/modules/metrics.css

```diff
@@ -0,0 +1,38 @@
+.metrics {
+    text-align: center;
+}
+
+.metrics--photo {
+    display: block;
+    width: 200px;
+    height: 200px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.metrics--photo > img {
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.metrics .sectionHeading {
```

Comment:
I think we should just use another class for this, instead of reusing `sectionHeading`. It is some duplication of the CSS rules, but I think it's worth it to reserve `sectionHeading` only for headings of page sections.

(As an example of why -- imagine if BAMP asks us to tweak the look of the metrics later. We'd want to be able to do that without impacting the section headings. Or vice versa.)